submissions/ScreenLimits/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "manifest_version": 3,
+  "name": "Screen Time Manager",
+  "version": "1.0",
+  "description": "Control your screen time across different websites",
+  "permissions": [
+    "storage",
+    "tabs",
+    "webNavigation",
+    "alarms",
+    "scripting"
+  ],
+  "host_permissions": [
+    "<all_urls>"
+  ],
+  "background": {
+    "service_worker": "background.js"
+  },
+  "action": {
+    "default_popup": "popup.html",
+    "default_icon": {
+      "16": "images/icon16.png",
+      "48": "images/icon48.png",
+      "128": "images/icon128.png"
+    }
+  },
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
+      "js": ["content.js"]
+    }
+  ],
+  "icons": {
+    "16": "images/icon16.png",
+    "48": "images/icon48.png",
+    "128": "images/icon128.png"
+  }
+}

```

Comment:
<img width="563" alt="image" src="https://github.com/user-attachments/assets/ef070ca4-5ff3-4f82-863b-16b060211732" />

It also appears the extension fails to load some icons